process-managers/pm2/run.sh

```diff
@@ -5,5 +5,5 @@ set -eux
 yarn install
 yarn prisma generate
 
-yarn pm2 start --name 'prisma-pm2' 'node server.js'
+yarn pm2 start --name 'prisma-pm2' 'node ./server.js'
```

Comment:
I would try this on Windows:
```suggestion
yarn pm2 start --name prisma-pm2 "node ./server.js"
```